### PR TITLE
output hierarchy and assessment data when doing xml_out

### PR DESCRIPTION
**Pull request by malaffoon**
If we want to disable the automatic creation of stub assessments, districts and schools during exam ingest, we need that data from the data generator. This is an initial quick go at that ...

### CODE REVIEW

File: data_generator/outputworkers/csv_star_worker.py [hunk 1]
```diff
@@ -23,15 +23,11 @@ def prepare(self):
         csv_writer.prepare_csv_file(sbac_out_config.DIM_ASMT_FORMAT['name'], sbac_out_config.DIM_ASMT_FORMAT['columns'], self.root_path)
         csv_writer.prepare_csv_file(sbac_out_config.SR_FORMAT['name'], sbac_out_config.SR_FORMAT['columns'], self.root_path)
 
-    def write_iab(self, asmt: Assessment):
+    def write_assessments(self, asmts: [Assessment]):
```
Comment (malaffoon) [inline data_generator/outputworkers/csv_star_worker.py]: Interestingly most of the legacy output workers want to get a collection of assessments, so i added this hook.

File: data_generator/outputworkers/csv_star_worker.py [hunk 2]
```diff
@@ -23,15 +23,11 @@ def prepare(self):
...
         file_name = sbac_out_config.DIM_ASMT_FORMAT['name']
-        csv_writer.write_records_to_file(file_name, sbac_out_config.DIM_ASMT_FORMAT['columns'], [asmt], tbl_name='dim_asmt', root_path=self.root_path)
+        csv_writer.write_records_to_file(file_name, sbac_out_config.DIM_ASMT_FORMAT['columns'], asmts, tbl_name='dim_asmt', root_path=self.root_path)
 
-    def write_assessment(self, asmt: Assessment):
-        file_name = sbac_out_config.DIM_ASMT_FORMAT['name']
-        csv_writer.write_records_to_file(file_name, sbac_out_config.DIM_ASMT_FORMAT['columns'], [asmt], tbl_name='dim_asmt', root_path=self.root_path)
-
-    def write_hierarchies(self, hierarchies: InstitutionHierarchy):
+    def write_hierarchies(self, hierarchies: [InstitutionHierarchy]):
```
Comment (malaffoon) [inline data_generator/outputworkers/csv_star_worker.py]: This was just wrong in the old code.

File: data_generator/writers/tabulator_writer.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
+"""
+A writer producing output like the assessment package tabulator.
+
+"""
+import csv
+
+TabulatorFieldNames = [
```
Comment (malaffoon) [inline data_generator/writers/tabulator_writer.py]: We don't have data for many of the columns which, of course, concerns me, especially the claim/target columns. However, the output does round-trip okay. I'm guessing there might be issues in the assessment package loader but i haven't gotten that far yet.

Comment (agorina) [inline data_generator/writers/tabulator_writer.py]: Hmmm...Claim and Target are ignored with my recent change, however it required a new field 'ClaimTarget'. Also, I believe it fails if some columns are missing and I do not see two new columns: OtherClaimTargets and CommonCore.

**Comment by agorina** [comment]
I like this change, this is way better than keeping an existing hack.

File: data_generator/writers/tabulator_writer.py (hunk 3 above)
Comment (malaffoon) [inline data_generator/writers/tabulator_writer.py]: Thanks, i'll add those columns. Of course, they'll be empty ...

**Pull request merged**